.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  specs:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: ['3.0', '2.7', '2.6', '2.5']
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+      - name: Install dependencies
+        run: bundle install
```

Comment:
@rahim do we need this? The `setup-ruby` should do this automatically:

```
        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

and https://github.com/ruby/setup-ruby/tree/v1#caching-bundle-install-automatically

Reply:
👍 Good catch, I'd not spotted that